src/ASM/Test/TestASMs3Dmx.C

```diff
@@ -0,0 +1,529 @@
+//==============================================================================
+//!
+//! \file TestASMs3Dmx.C
+//!
+//! \date Aug 25 2023
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Tests for structured 3D mixed spline FE models.
+//!
+//==============================================================================
+
+#include "ASMCube.h"
+
+#include "gtest/gtest.h"
+
+
+namespace {
```

Comment:
Same namespace question applies here too. There is probably a good reason, I just don't see it.